src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/FilterColumnsWithAllNullValues.java

```diff
@@ -0,0 +1,37 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.foundations.file.LocalFile;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Column;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Dataset;
+import com.northconcepts.datapipeline.foundations.pipeline.input.ExcelPipelineInput;
+
+public class FilterColumnsWithAllNullValues {
+
+    public static void main(String[] args) {
+
+        Pipeline pipeline = new Pipeline();
+
+        LocalFile inputFile = new LocalFile()
+                .setName("Input File")
+                .setPath("PATH_TO_INPUT_FILE.csv")
```

Comment:
Let's create a working example and read from https://github.com/NorthConcepts/DataPipeline-Examples/tree/master/data/input like we do in other examples.

Reply:
done